frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -664,6 +714,11 @@ export const ClusterServiceVersionsPage: React.FC<ClusterServiceVersionsPageProp
             namespace: props.namespace,
             prop: 'clusterServiceVersions',
           },
+          {
+            kind: referenceForModel(ClusterServiceVersionModel),
+            namespace: 'openshift-operators',
+            prop: 'allClusterServiceVersions',
+          },
```

Comment:
This is going to break basic users who can't list CSVs in this namespace.

@zherman0 Can you expand on why we need to this now?

Reply:
Yes, it does.  This needs to be reworked.